src/data/markdown/.eslintrc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 7,
```

Comment:
Change this to `ecmaVersion: 8` and you shouldn't need `requireConfigFile: false`.

7 version does not recognise `async/await` syntax, although it's pretty weird that it notifies such things as a `babel` config error 🤷

Reply:
Yeah that is true - I have now redone the whole PR